internal/pkg/istio-router-check/cmd/root.go

```diff
@@ -172,7 +172,7 @@ func (c *RootCommand) prepareTests(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("could not read directory %s: %w", c.TestDir, err)
 	}
-	testCases, err := parser.ParseTestCases(oldFiles)
+	testCases, err := parser.ParseTestCases(oldFiles, true)
```

Comment:
nit: passing a plain bool means you need to look at the call to understand what's happening.